### PR TITLE
Coinify limits

### PR DESCRIPTION
**Pull request by jstettner**

### CODE REVIEW

**Comment by plondon** [comment]
Looks really good so far! I think we could improve the user experience a bit. I tried clicking on the min/max numbers in the error, assuming it would populate the field. We do this in send for example. Do you think we could do it here as well?

File: app/templates/buy-quick-start.pug [hunk 1]
```diff
@@ -54,8 +54,8 @@
               button.btn.btn-default.brdrn
                 span.btc BTC
         .mts.pos-rel(ng-hide="tradingDisabled")
-          span.pos-abs.type-sm.state-danger-text(ng-show="fiatForm.fiat.$error.min" translate="BUY_AMT_LOW" translate-values="{limit: limits.absoluteMin(transaction.currency.code), symbol: symbol}")
-          span.pos-abs.type-sm.state-danger-text(ng-show="fiatForm.fiat.$error.max" translate="BUY_AMT_HIGH" translate-values="{limit: limits.absoluteMax(transaction.currency.code), symbol: symbol}")
+          span.pos-abs.type-sm.state-danger-text(ng-click="setFiat(limits.absoluteMin(transaction.currency.code)); updateLastInput('fiat'); transaction.fiat && getQuote();" ng-show="fiatForm.fiat.$error.min" translate="BUY_AMT_LOW" translate-values="{limit: limits.absoluteMin(transaction.currency.code), symbol: symbol}")
+          span.pos-abs.type-sm.state-danger-text(ng-click="setFiat(limits.absoluteMax(transaction.currency.code)); updateLastInput('fiat'); transaction.fiat && getQuote();" ng-show="fiatForm.fiat.$error.max" translate="BUY_AMT_HIGH" translate-values="{limit: limits.absoluteMax(transaction.currency.code), symbol: symbol}")
```
Comment (plondon) [inline app/templates/buy-quick-start.pug]: > updateLastInput('fiat'); transaction.fiat && getQuote();

Is this necessary? I think corresponding input fields watch for changes and subsequently run these functions.

File: assets/js/components/sell-quick-start.component.js [hunk 2]
```diff
@@ -108,6 +108,11 @@ function sellQuickStartController ($scope, $rootScope, currency, buySell, Alerts
     }
   };
 
+  $scope.setFiat = (amount) => {
+    console.log(amount);
```
Comment (plondon) [inline assets/js/components/sell-quick-start.component.js]: Please remove console log

File: app/templates/buy-quick-start.pug (hunk 1 above)
Comment (jstettner) [inline app/templates/buy-quick-start.pug]: it doesn't update without those calls. i guess setting the model directly doesn't trigger ng-change

**Pull request merged**